From ff97ff3982e068e07591830aa68fbc25eac1b7ee Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Wed, 2 Aug 2017 04:00:33 -0500
Subject: [PATCH 2320/5242] MLK-16126 arm64: dts: imx8qm: Fix pd_sata0 node's
 name

commit  a036909587ef3402302fac8ab0506309c81439e0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Should use PD_HSIO_SATA_0 not the PD_HSIO_SATA0 to align with imx8_pd.h.

Signed-off-by: Ye Li <ye.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 12b3c80..e718703 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -514,7 +514,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 
-				pd_sata0: PD_HSIO_SATA0 {
+				pd_sata0: PD_HSIO_SATA_0 {
 					reg = <SC_R_SATA_0>;
 					#power-domain-cells = <0>;
 					power-domains =<&pd_serdes1>;
-- 
1.7.9.5

